generators/route/index.js

```diff
@@ -196,9 +196,9 @@ module.exports = class extends Generator {
           name: nameAnswer,
           componentName: name,
           airbnbLinting: this.answers.airbnbLinting,
-          startCasedName,
-          camelName: lowerFirst(startCasedName.replace(/ /g, '')),
-          pathName: startCasedName.toUpperCase().replace(/ /g, '_')
+          startCaseName,
```

Comment:
Normally I would have fixed the instance in the template, but I think the change you made is actually more clear, nice work!